database/migrations/2017_05_19_125403_create_diagnostic_report_tables.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateDiagnosticReportTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        //Based on FHIR - https://www.hl7.org/fhir/diagnosticreport.html
+        Schema::create('panels', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('panel_type_id')->unsigned();
+            $table->integer('performed_by')->unsigned();//User who performed this report
+            $table->integer('specimen_id')->unsigned(); // Specimens this report is based on
+            $table->string('conclusion'); // Clinical Interpretation of test results
+            $table->integer('coded_diagnosis')->unsigned(); // Codes for the conclusion
+            $table->integer('status_id')->unsigned();//Active\Inactive
+            $table->integer('sort_order');
+                    
+            //Several timestamps pending/ also responsible actors for various activites
+            $table->timestamps();
+            $table->softDeletes();
+
+            $table->foreign('performed_by')->references('id')->on('users');
+            $table->foreign('specimen_id')->references('id')->on('specimens');
+            $table->foreign('coded_diagnosis')->references('id')->on('codeable_concepts');
```

Comment:
Wasn't thus suppose to be something like codeable_concept_id for the sake of relationships? Or do we keep it that way for easy understanding?